system_files/shared/usr/bin/sb-key-notify

```diff
@@ -0,0 +1,9 @@
+KEY_WARN_FILE="/run/user-motd-sbkey-warn.md"
+
+if [ -e $KEY_WARN_FILE ]; then
+    WARNING_MSG=$(cat "$KEY_WARN_FILE")
+
+    notify-send "WARNING" "$(echo "$WARNING_MSG" | tr -d '*~')" -i dialog-warning -u critical -a mokutil --wait
+else
+    echo "No secure boot key warning file.  Skipping...
```

Comment:
Missing a closing quote after `Skipping...`

Results in this error when the script runs and they key is not enrolled:

```
❯ /usr/bin/sb-key-notify 
/usr/bin/sb-key-notify: line 8: unexpected EOF while looking for matching `"'
```